service/history/transferQueueActiveTaskExecutor.go

```diff
@@ -1303,6 +1303,7 @@ func (t *transferQueueActiveTaskExecutor) resetWorkflow(
 		),
 		reason,
 		nil,
+		ResetReapplyTypeAll,
```

Comment:
Can you remind me again why we have Reset task for transfer task processing? 

Is this for the case where we don't perform reset synchronously and instead just create transfer task and actual reset processing happens as part of transfer task processing.

What should be the standby equivalent in this case?

Reply:
i guess the answer is auto reset points.

> What should be the standby equivalent in this case?

standby should do nothing, standby should only follow active's order